api/v1alpha1/openapi.yaml

```diff
@@ -2022,7 +2022,7 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/EnrollmentRequest'
+                $ref: '#/components/schemas/Error'
```

Comment:
POSTs (Create) of resources should only have 201, 400, 401, 403, 409, 503 responses (see the linting rules in [PR #714](https://github.com/flightctl/flightctl/pull/714)).

The correct code here would be 409, as the resource being created already exists. We could also remove the 200, 202, and 208 while we're at it.